lsof.yaml

```diff
@@ -0,0 +1,70 @@
+package:
+  name: lsof
+  version: 4.99.3
+  epoch: 0
+  description: List Open Files
+  copyright:
+    - license: lsof
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - bash
+      - build-base
+      - busybox
+      - groff
+      - libtool
+      - linux-headers
+      - openssf-compiler-options
+      - pkgconf-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/lsof-org/lsof
+      tag: ${{package.version}}
+      expected-commit: 2e4c7a1a9bc7258dc5b6a3ab28ebca44174279a8
+
+  - runs: autoreconf -vif
+
+  - uses: autoconf/configure
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+  - uses: strip
+
+subpackages:
+  - name: lsof-dev
+    description: dev library for lsof
+    dependencies:
+      runtime:
+        - lsof
+    pipeline:
+      - uses: split/dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
+
+  - name: lsof-doc
+    pipeline:
+      - uses: split/manpages
+    description: lsof manpages
+
+  - name: lsof-debug
+    pipeline:
+      - uses: split/debug
+    description: lsof debug
+
+update:
+  enabled: true
+  github:
+    identifier: lsof-org/lsof
+
+test:
+  pipeline:
+    - name: version test
+      runs: /usr/bin/lsof -v
```

Comment:
One thing that's not working as expected for now is that, this version is wrong. It returns 4.99.0 whereas we are building for 4.99.3